programs/fileio.c

```diff
@@ -388,6 +388,12 @@ static FILE* FIO_openSrcFile(const char* srcFileName)
         return stdin;
     }
 
+    if (!UTIL_fileExist(srcFileName)) {
+        DISPLAYLEVEL(1, "zstd: %s : No such file or directory (can't stat) -- ignored \n",
```

Comment:
I think we should print

```c
DISPLAYLEVEL(1, "zstd: %s: %s -- ignored \n", srcFileName, strerror(errno));
```

If the file doesn't exist the strerror is `No such file or directory`.

Reply:
I wasn't sure if it would work properly after `stat()`, but indeed, it does !
Fixed